src/igvfd/schemas/tabular_file.json

```diff
@@ -121,15 +121,39 @@
                 "peaks",
                 "prime editing guide RNA sequences",
                 "protein to protein interaction score",
-                "tissue positions",
                 "sample sort parameters",
                 "sequence barcodes",
                 "SNP effect matrix",
+                "tissue positions",
                 "variants",
                 "variant binding effects",
                 "variant effects",
                 "variant to element mapping"
             ],
+            "enum_descriptions": {
```

Comment:
I would replace "Mapping of... " with "A File describing the mapping of ..." for many of these

Reply:
fixed